README.md

```diff
@@ -86,25 +68,6 @@ AWS_REGION: us-east-1
 SSL_CERT: /tmp/ssl-cert.pem
 ```
 
-#### Push with `meta-buildpack`
-
-Before running `cf push`, ensure that:
-
-+ `meta-buildpack` is installed and at the top of list of buildpacks
-+ No buildpack is specified in your application's manifest
-+ You will not specify a buildpack with the `-b` flag on `cf push`
-
-If any of these conditions are not met, follow the [alternate usage directions](#custom-buildpack-usage) below.
-
-Once you run `cf push`, assuming the app is bound to a Conjur service instance, the `meta-buildpack` will first invoke the appropriate language buildpack and then any decorator buildpacks (including the Conjur Buildpack). The secrets specified in the `secrets.yml` file will be available in the session environment variables at the start of the app.
-
-#### <a name="custom-buildpack-usage"></a>Usage for users with custom buildpacks
-
-To retrieve secrets without using `meta-buildpack` you can simply:
-
-+ copy the contents of the `./lib` directory in a release to the `./.profile.d` directory relative to your app's root directory.
-+ copy the contents of the `./vendor` directory in a release to the `./vendor` directory relative to your app's root directory.
-
 When your application starts the secrets specified in the `secrets.yml` file will now be available in the session environment variables at the start of the app.
 
```

Comment:
We may still have users who need to use the decorator buildpack functionality, and they can do so using the 1.0.0 release of the buildpack. I feel pretty strongly that we should continue to include those instructions somewhere - can we either do that here, or log an issue in the [conjur-docs](https://github.com/cyberark/conjur-docs) to add documentation for using the original meta-buildpack functionality to the (non-existent yet) CF buildpack documentation?

Reply:
Issue #21 will address this